docs/design-documents/tools/cmake.md

```diff
@@ -0,0 +1,116 @@
+# CMake Mbed OS
```

Comment:
Same a pointer to the background and why/what would be helpful

Reply:
We will update this